stubs/pyinstaller/pyi_splash/__init__.pyi

```diff
@@ -8,5 +8,5 @@ def is_alive() -> bool: ...
 def update_text(msg: str) -> None: ...
 def close() -> None: ...
 
-CLOSE_CONNECTION: Literal[b"\u0004"]
+CLOSE_CONNECTION: Literal[rb"\u0004"]
```

Comment:
```suggestion
CLOSE_CONNECTION: Literal[b"\x04"]
```
I think what you have would be the wrong byte string. It's just `b'\x04'`:
https://github.com/pyinstaller/pyinstaller/blob/develop/PyInstaller/fake-modules/pyi_splash.py#L69

Reply:
Huh, I'm surprised stubtest didn't pick up on that.